test/train-sets/ref/cats_save.stderr

```diff
@@ -15,17 +15,17 @@ power_t = 0.5
 Enabled reductions: ftrl-Coin Betting, generate_interactions, scorer-identity, binary, cats_tree, get_pmf, pmf_to_pdf, cb_explore_pdf, cats_pdf, sample_pdf, cats
 Input label = continuous
 Output pred = action_pdf_value
-average  since         example        example        current        current  current 
-loss     last          counter         weight          label        predict features 
-0.000000 0.000000            1            1.0 {0.00120175... 0.00120175,...        6 
-0.000000 0.000000            2            2.0 {0.582045,0... 0.611352,0....        6 
-0.000000 0.000000            4            4.0 {10.3501,0,... 2.23432,0.4...        6 
-0.000000 0.000000            8            8.0 {2.40591,0,... 22.5925,0.0...        6 
+average  since         example        example        current        current  current
+loss     last          counter         weight          label        predict features
+0.000000 0.000000            1            1.0     {0,0,0.01}         0,0.01        6
```

Comment:
just an option to consider:
make formatting related to number of significant bits to avoid having pure 0 - it may be misleading in situation like this since pure 0 float somehow gives impression of variable that have never been changed

Reply:
Which column are you referring to? The left two I didn't change, they used fixed formatting of 7 (think) precision. The approach I went for for the other numbers is 2 decimal places of precision and any trailing zeroes removed.

Reply:
I mean situation when 0.00120175 turns into 0.
It may be useful to output it either 1.2e-3 or atleast something like "0..."

Reply:
but it is nit for sure

Reply:
Yeah that thing is tricky.  For 0.00120175 like you said with only two decimal places of precision it just becomes 0. However if you use significant digits then for large numbers you lose too much information such as 1234 becomes 1200 which seems like a pretty significant loss of into. The medium I landed on was to support two decimal places of precision and all digits left of the decimal.

Reply:
Yeah I see. I think it is ok to do it this way.